app/models/gql/Objects.scala

```diff
@@ -203,6 +205,12 @@ object Objects extends Logging {
                        Some("Disease children entities in ontology"),
                        resolve = r => diseasesFetcher.deferSeq(r.value.children))),
     AddFields(
+      Field(
+        "locations",
+        OptionType(ListType(diseaseImp)),
```

Comment:
Does this need to be `OptionType(ListType(...))` or can it just be `ListType`?

Reply:
Done